create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+3. A git client has to be installed as well depending on the distro.
+  
+    E.g.: on MacOS it can be installed via Homebrew:
+    ```bash
+    brew info git
+    brew install git
+    ```
+
+4. [Terraform](https://www.terraform.io/downloads.html) must be installed next. 
+    
+    E.g.: on MacOS it can be installed via Homebrew:
+    ```bash
+    brew info terraform
+    brew install terraform
+    ```
+
+### Installation steps
+
+1. Start by creating a [Github personal access token](https://github.com/settings/tokens) that has the `repo` permissions set. The Terraform module provided by this blueprint will automatically create for us a Github repository and branch where all the required Kubernetes manifests will be stored. In order to achieve this we must create a few special environment variables so make sure to replace the `<>` placeholders accordingly:
+
+    ```bash
+    export TF_VAR_github_owner="<github_username>"                              # should be pretty self explanatory
+    export TF_VAR_github_token="<github_personal_access_token>"                 # should be pretty self explanatory
+    export TF_VAR_github_repository_name="<git_repository_name>"                # the desired git repository name
+    export TF_VAR_github_repository_branch="<git_repository_branch>"            # branch name to be created
+    export TF_VAR_github_repository_target_path="<flux_cd_sync_target_path>"    # the K8S manifests used to provision Flux CD will be stored in this path
+    ```
+2. Let's go next with setting up the required stuff on the [DigitalOcean](https://cloud.digitalocean.com) side by creating the necessary tokens first. One is needed for creating/managing the DOKS cluster and another one for [DO Spaces](https://cloud.digitalocean.com/spaces) (S3-compatibe object storage). The latter is needed for storing the Terraform state file.
+
+    From your DigitalOcean account go to the [API](https://cloud.digitalocean.com/account/api) section in order to generate the tokens. 
+    
+    Create a personal access token first by clicking the `Generate New Token` button:
+
+    ![Applications & API Key](content/img/pt_gen_1st.png)
+
+    then make sure to give it a proper name and that the `Write` scope is checked:
+
+    ![Applications & API Key](content/img/pt_gen_2nd.png)
+
+    In the end we should have something like this:
+
+    ![Applications & API Key](content/img/api_access_key.png)
+
+    Copy the token value and save it in a local environment variable as we will need it later on (make sure to replace the `<>` placeholder):
+
+    ```bash
+    export TF_VAR_do_token="<your_do_personal_access_token>"
+    ```
+    Next we navigate to `Spaces access keys` and click the `Generate New Key` button:
+
+    ![Spaces Key](content/img/spk_gen_1st.png)
+
+    then give it a proper name and mark it as done:
+
+    ![Spaces Key](content/img/spk_gen_2nd.png)
+
+    In the end we should have something like this:
```

Comment:
```suggestion
    In the end, we should have something like this:
```

Reply:
Thanks for the suggestion. Fixed in the latest commit.